content/smart.md

```diff
@@ -137,10 +137,6 @@ For provider and patient facing apps, you can select any patient from a list of
 
 Once you have your app working in code Console and you are committed to taking your app to production, contact us through the application available at [https://code.cerner.com/millennium](https://code.cerner.com/millennium), and we can provide access to *PowerChart* to test your app. For provider facing apps, PowerChart is the environment where providers will access apps in a production environment, so it is very important to test the apps in *PowerChart*. Access to PowerChart is a benefit only offered to members of our code program.
 
-### HealtheLife ###
```

Comment:
If the removal of the HealtheLife section was intended, it would be best on a separate PR as this PR was documented as being specific to the port documentation below